test/e2e-tests.sh

```diff
@@ -35,4 +37,30 @@ header "Running Go e2e tests"
 go_test_e2e ./test || failed=1
 
 (( failed )) && fail_test
+
+go build github.com/tektoncd/cli/cmd/tkn
+
+./tkn pipeline list && fail_test "Failed to list pipelines"
+./tkn pipeline describe foo && fail_test "Failed to describe pipeline"
+#./tkn pipelinerun list && fail_test "Failed to list pipelinerun"
+./tkn pipelinerun describe foo && fail_test "Failed to describe pipelinerun"
+#./tkn pipelinerun logs foo && fail_test "Failed to show logs"
+
+install_pipeline_crd
+
+# change default namespace to tektoncd
+kubectl create namespace tektoncd
+kubectl config set-context $(kubectl config current-context) --namespace=tektoncd
+
+# create pipeline, pipelinerun, task, and taskrun
+kubectl apply -f ./resources/output-pipelinerun.yaml
```

Comment:
I am guessing the working dir is the git root, so it should be `./test/resource/…` for those